style.css

```diff
@@ -302,3 +302,69 @@ button:disabled {
   border-radius: 0;
   color: #000;
 }
+
+.contacts {
+  background-image: url("./images/contactshapes.png");
+  display: flex;
+  background-color: #396df2;
+  flex-direction: column;
+  justify-content: space-around;
+  width: 100%;
+  background-repeat: no-repeat;
+  background-position: right;
+  background-size: 90% 100%;
+  border-top-left-radius: 100px;
+}
+
+.contact-section {
+  display: flex;
+  flex-direction: column;
+  width: 90%;
+  gap: 10px;
+  margin: 0 auto 50px;
+  background-repeat: no-repeat;
+}
+
+.contact-intro {
+  margin-top: 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contact-me {
+  font-family: "poppins", sans-serif;
+  margin-block-end: 0;
+  font-size: 40px;
+  font-weight: 900;
```

Comment:
- [ ] Please see the font-weight property according to the figma template is 700, Please fix this too 

![image](https://user-images.githubusercontent.com/97521557/212975216-917273b0-67dc-4991-9d61-d600416abe76.png)